<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cow_Feeding_Monitoring</class>
 <widget class="QMainWindow" name="Cow_Feeding_Monitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cow_Feeding_Monitoring</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main Screen</string>
     </attribute>
     <widget class="QLabel" name="Eating_Img_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Leaving_Img_label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>450</width>
        <height>301</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>19</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Feeding</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>19</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Leaving</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Fence Test</string>
     </attribute>
     <widget class="QLabel" name="Capture_Img_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>450</width>
        <height>300</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Detect_Img_label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>450</width>
        <height>300</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>961</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>System Log</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>941</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>261</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="webcam_radioButton">
         <property name="text">
          <string>WebCam</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="picamera_radioButton">
         <property name="text">
          <string>Pi Camera</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDoubleSpinBox" name="Threshold_spinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <double>0.200000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Option</string>
     </attribute>
     <widget class="QRadioButton" name="vertical_flip_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>vertical_flip</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="recx0">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recx1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>275</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recx2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>540</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recx3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>840</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>w</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="recwidth0">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>260</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recwidth1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>260</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recwidth2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>275</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recwidth3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="recy">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="recheight">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>560</number>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="Capture_Btn">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Test_rec_pos">
         <property name="text">
          <string>Test_ROI</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Fence_par_ok_Btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="scale_factor_doubleSpinBox">
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>vertical_flip_radioButton</zorder>
     <zorder>label</zorder>
     <zorder>recx0</zorder>
     <zorder>recx1</zorder>
     <zorder>recx2</zorder>
     <zorder>recx3</zorder>
     <zorder>label_4</zorder>
     <zorder>recwidth0</zorder>
     <zorder>recwidth1</zorder>
     <zorder>recwidth2</zorder>
     <zorder>recwidth3</zorder>
     <zorder>recy</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>recheight</zorder>
     <zorder>Fence_par_ok_Btn</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1253</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
